GameOfHive/ViewController.swift

```diff
@@ -191,7 +185,10 @@ extension ViewController {
         if motion == .MotionShake {
             stop()
             dispatch_async(gridQueue) {
-                let grid = emptyGrid(self.numberOfRows,columns: self.numberOfColumns)
+                let cellWidth = CGFloat(22)
+                let cellHeight = CGFloat(25)
```

Comment:
Deze waarden zijn ook in cellSize gedefinieerd, dus kunnen we ze hier niet beter uithalen?